target/linux/generic/files/drivers/platform/mikrotik/rb_softconfig.c

```diff
@@ -290,6 +291,11 @@ static struct sc_boolts const sc_booter = {
 	.strtrue = "backup",
 };
 
+static struct sc_boolts const sc_silent_boot = {
+	.strfalse = "off",
+	.strtrue = "on",
+};
+
```

Comment:
After the rename to `silent_boot`, this should be reversed I think?

Silent boot On = Buzzer off
Silent boot Off = Buzzer on

Reply:
And that's exactly what it does (or should do) :)
I reversed the previous logic if you compare with https://github.com/openwrt/openwrt/blob/1389e4972bcff5e281106be132760d23ee305384/target/linux/generic/files/drivers/platform/mikrotik/rb_softconfig.c#L294-L297